supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesResponseJsonFormatter.scala

```diff
@@ -2,24 +2,19 @@ package cromwell.backend.impl.tes
 
 import spray.json._
 
-final case class TesPostResponse(value: String)
-
-final case class TesGetResponse(
-                                 jobID: String,
-                                 task: TesTaskMessage,
-                                 state: String,
-                                 logs: Option[Seq[JobLogs]]
-                               )
+final case class CreateTaskResponse(id: String)
 
+final case class CancelTaskResponse()
 
 object TesResponseJsonFormatter extends DefaultJsonProtocol {
-  implicit val volumeFormat = jsonFormat5(Volume)
   implicit val resourcesFormat = jsonFormat5(Resources)
   implicit val taskParameterFormat = jsonFormat6(TaskParameter)
   implicit val portsFormat = jsonFormat2(Ports)
-  implicit val dockerExecutorFormat = jsonFormat7(DockerExecutor)
-  implicit val tesTaskMessageFormat = jsonFormat7(TesTaskMessage)
-  implicit val jobLogsFormat = jsonFormat8(JobLogs)
-  implicit val tesPostResponseFormat = jsonFormat1(TesPostResponse)
-  implicit val tesGetResponseFormat = jsonFormat4(TesGetResponse)
+  implicit val executorFormat = jsonFormat8(Executor)
+  implicit val executorLogFormat = jsonFormat7(ExecutorLog)
+  implicit val outputFileLogFormat = jsonFormat3(OutputFileLog)
+  implicit val taskLogFormat = jsonFormat5(TaskLog)
+  implicit val taskFormat = jsonFormat12(Task)
+  implicit val createTaskResponseFormat = jsonFormat1(CreateTaskResponse)
+  implicit val cancelTaskResponseFormat = jsonFormat0(CancelTaskResponse)
```

Comment:
Just curious, what JSON object ends up these internal messages in them? It seems unlikely a user would ever want to see them